### ISSUE TITLE
pcntl_fork

### ISSUE THREAD

**Issue by larcho** [sobstel/metaphore#17]
Have you considered implementing pcntl_fork for the callback functions? That might avoid freezing the callback request to the client that currently is "unstaling" the result.

**Comment by sobstel** [comment]
No, never considered it for my use case to be honest. Actually I don't feel strong with `pcntl` extension at all and I'm not even sure if it would be possible with current lib and callback design.
